tests/regression

```diff
@@ -786,10 +794,7 @@ test_sync() {
         --log-dir ./logs
     wait_for_socket
 
-    # kill running indexer and release db lock
-    kill "$(cat idxr_pid)" 2>/dev/null || true
-    rm -f idxr_pid
-    rm ./database/LOCK
```

Comment:
Do we need to worry about the `./database/LOCK` anymore?

Reply:
The explicit removal of the lock file was only needed because we were silently failing to terminate the Mina Indexer process (2 lines above). This PR attempts to correct the termination, which achieves the deletion of the lock file.